File: petstore/java/src/main/java/oseg/petstore_examples/CreateUsersWithListInputExample.java
package oseg.petstore_examples;

import org.openapitools.client.ApiException;
import org.openapitools.client.Configuration;
import org.openapitools.client.api.*;
import org.openapitools.client.auth.*;
import org.openapitools.client.model.*;

import java.io.File;
import java.time.LocalDate;
import java.time.OffsetDateTime;
import java.util.ArrayList;
import java.util.List;
import java.util.Map;

public class CreateUsersWithListInputExample
{
    public static void main(String[] args)
    {
        var config = Configuration.getDefaultApiClient();
        config.setApiKey("YOUR_API_KEY");

        var user1 = new User();
        user1.id(12345L);
        user1.username("my_user_1");
        user1.firstName("John");
        user1.lastName("Doe");
        user1.email("john@example.com");
        user1.password("secure_123");
        user1.phone("555-123-1234");
        user1.userStatus(1);

        var user2 = new User();
        user2.id(67890L);
        user2.username("my_user_2");
        user2.firstName("Jane");
        user2.lastName("Doe");
        user2.email("jane@example.com");
        user2.password("secure_456");
        user2.phone("555-123-5678");
        user2.userStatus(2);

        var user = new ArrayList<User>(List.of (
            user1,
            user2
        ));

        try
        {
            new UserApi(config).createUsersWithListInput(
                user
            );
        } catch (ApiException e) {
            System.err.println("Exception when calling User#createUsersWithListInput");
            System.err.println("Status code: " + e.getCode());
            System.err.println("Reason: " + e.getResponseBody());
            System.err.println("Response headers: " + e.getResponseHeaders());
            e.printStackTrace();
        }
    }
}

File: petstore/kotlin/src/main/kotlin/oseg/petstore_examples/CreateUsersWithListInputExample.kt
package oseg.petstore_examples

import org.openapitools.client.infrastructure.*
import org.openapitools.client.apis.*
import org.openapitools.client.models.*

import java.io.File
import java.time.LocalDate
import java.time.OffsetDateTime
import kotlin.collections.ArrayList
import kotlin.collections.List
import kotlin.collections.Map
import com.squareup.moshi.adapter

@ExperimentalStdlibApi
class CreateUsersWithListInputExample
{
    fun createUsersWithListInput()
    {
        ApiClient.apiKey["api_key"] = "YOUR_API_KEY"

        val user1 = User(
            id = 12345,
            username = "my_user_1",
            firstName = "John",
            lastName = "Doe",
            email = "john@example.com",
            password = "secure_123",
            phone = "555-123-1234",
            userStatus = 1,
        )

        val user2 = User(
            id = 67890,
            username = "my_user_2",
            firstName = "Jane",
            lastName = "Doe",
            email = "jane@example.com",
            password = "secure_456",
            phone = "555-123-5678",
            userStatus = 2,
        )

        val user = arrayListOf<User>(
            user1,
            user2,
        )

        try
        {
            UserApi().createUsersWithListInput(
                user = user,
            )
        } catch (e: ClientException) {
            println("4xx response calling UserApi#createUsersWithListInput")
            e.printStackTrace()
        } catch (e: ServerException) {
            println("5xx response calling UserApi#createUsersWithListInput")
            e.printStackTrace()
        }
    }
}
